.github/workflows/container-lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - webv2
+    paths:
+      - 'Dockerfile'
+  pull_request:
+    branches:
+      - webv2
+    paths:
+      - 'Dockerfile'
+
+jobs:
+  trivy:
+    name: Dockerfile
+    runs-on: ubuntu-latest
+    container:
+      image: aquasec/trivy
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check critical issues
+        run: trivy config --exit-code 1 --severity "HIGH,CRITICAL" ./Dockerfile
```

Comment:
What's trivy?

Reply:
I don’t have any specific preferences. Which dockerfile checker you want to us?

Reply:
I was just curious what trivy was. So it's a linter for Dockerfiles? Does it support Earthfiles too?

Reply:
it is a security scanner https://aquasecurity.github.io/trivy/v0.35/ Other than docekrfiles, it supports many other formats but unfortunately it does not support Earthfiles yet.
It can detect if we use root as the user inside the container (the main issue addressed by this PR). It also had some recommendations for the dockerfile which I applied. See the report for non-critical issues [here]( https://github.com/owncast/owncast/actions/runs/3776695073/jobs/6420111585)